public/api-spec.json

```diff
@@ -19,46 +19,23 @@
   },
   "basePath": "/api",
   "paths": {
-    "/access-control/builtin-roles": {
-      "get": {
-        "description": "You need to have a permission with action `roles.builtin:list` with scope `roles:*`.",
-        "tags": ["access_control", "enterprise"],
-        "summary": "Get all built-in role assignments.",
-        "operationId": "listBuiltinRoles",
-        "responses": {
-          "200": {
-            "$ref": "#/responses/listBuiltinRolesResponse"
-          },
-          "403": {
-            "$ref": "#/responses/forbiddenError"
-          },
-          "500": {
-            "$ref": "#/responses/internalServerError"
-          }
-        }
-      },
+    "/admin/ldap/reload": {
```

Comment:
You need to regenerate with Enterprise 😄

Reply:
I did :shrug: You'll have to teach me how to do it properly, didn't find any proper instructions.